Source/CMakeLists.txt

```diff
@@ -82,6 +94,7 @@ set(WOOF_SOURCES
     r_things.c r_things.h
     s_sound.c  s_sound.h
     s_musinfo.c s_musinfo.h
+    sha1.c     sha1.h
```

Comment:
What do wen need this for? Do we even need this at all?

Reply:
Currently, we don't use it. I guess I should remove it.